Hush/Assets/Scripts/Weapon/InvisibleSpell.cs

```diff
@@ -18,15 +18,12 @@ public class InvisibleSpell : MonoBehaviour, ISpell
         [SerializeField] private float speedBoost = 1f;
         
         private static InputManager Input => InputManager.Instance;
-
         private PlayerMovement _player;
         
-        //TODO Remove
-        private void Awake()
-        {
-            // GameManager.Instance.SetActiveHeavySpell(SpellType.InvisibleSpell);
-            // GameManager.Instance.SetActiveLightSpell(SpellType.InvisibleSpell);
-        }
+        public SpellType SpellType => SpellType.InvisibleSpell;
+        
+        public float HeavyCooldown => 10f;
+        public float LightCooldown => 6f;
```

Comment:
Probably want those as SerializeField for all spells

Reply:
[SerializeField] private float heavyCooldown = 10f;

public float HeavyCooldown => heavyCooldown;